nixos/modules/system/boot/tmp.nix

```diff
@@ -30,7 +30,13 @@ with lib;
 
   config = {
 
-    systemd.additionalUpstreamSystemUnits = optional config.boot.tmpOnTmpfs "tmp.mount";
+    systemd.mounts = mkIf config.boot.tmpOnTmpfs [
+      {
+        what = "tmpfs";
+        where = "/tmp";
+        mountConfig.Options = [ "mode=1777" "strictatime" "rw" "nosuid" "nodev" "size=50%" ];
```

Comment:
The commit log seems to need to be adjusted as there is no huge page support here.  This seems to have been forgotten to be reworked when changing the commit.

Reply:
The commit message doesn't contain any reference to huge page support anymore.